deploy.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+export $(grep -v '^#' ".env" | gxargs -d '\n')
+
+if [[ "$#" -lt 1 ]]; then
+    echo "Usage: ./deploy.sh <service-name>"
+    exit 1
+fi
+
+SERVICE_NAME=$1
+
+echo "Pulling $SERVICE_NAME..."
+# shellcheck disable=SC2029
+ssh -p "$DEPLOY_PORT" "$DEPLOY_USER@$DEPLOY_HOST" "cd $DEPLOY_FOLDER && ./helper.sh --pull $SERVICE_NAME"
+
+echo "(Re)starting $SERVICE_NAME..."
```

Comment:
we could use directly this target (after changing it) https://github.com/AEGEE/oms-docker/blob/master/helper.sh#L244

we have to change it because it still has the former "we build on prod" mentality, tho that is the original purpose of the target

Reply:
yeah, but what  propose is more granular, it gives your more control on what you are  actually doing. Imagine somebody pulling  a container and not recreating it, then somebody trying to pull another container, then it'll recreate both of them and the deployer will be like "wtf just happened".

I'd say we should go for more granular commands and only deploy what's needed.